ggml/src/CMakeLists.txt

```diff
@@ -504,15 +512,14 @@ if (GGML_SYCL)
     list(APPEND GGML_SOURCES_SYCL "ggml-sycl.cpp")
 
     if (WIN32)
+        find_package(IntelSYCL REQUIRED)
         set(GGML_EXTRA_LIBS ${GGML_EXTRA_LIBS} IntelSYCL::SYCL_CXX MKL::MKL MKL::MKL_SYCL)
     else()
-        add_compile_options(-I/${SYCL_INCLUDE_DIR})
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsycl -L${MKLROOT}/lib")
```

Comment:
`-L${MKLROOT}/lib` is this flag not needed due to `find_package(MKL REQUIRED)` or is it missing?

Reply:
Yeah `find_package` does what's needed I believe.